data/mods/Tanks/dummied_out.json

```diff
@@ -0,0 +1,1010 @@
+[
```

Comment:
Is this spawned anywhere? If not, we are not going to drag around thousands of lines of json that doesn't actually do anything.  If it is, it needs to be renamed appropriately.